code/game/machinery/rechargestation.dm

```diff
@@ -269,3 +269,16 @@
 	if(!usr.incapacitated())
 		return
 	go_in(usr)
+
+/obj/machinery/recharge_station/MouseDrop_T(var/mob/target, var/mob/user)
```

Comment:
Hey  i wrote this feature on aurora :D didja copy it?

Anyways make it slower. About 15-20 seconds, but reduced by the user's robustness stat

Reply:
nope, i made it myself, i adapted it to Eris tho (added duration)